GVRf/Framework/framework/src/main/jni/objects/components/render_data.cpp

```diff
@@ -36,9 +36,14 @@ const RenderPass* RenderData::pass(int pass) const {
 }
 
 void RenderData::set_mesh(Mesh* mesh) {
-    mesh_ = mesh;
-    mesh->add_dirty_flag(dirty_flag_);
-    *dirty_flag_ = true;
+    if (mesh != nullptr) {
```

Comment:
What is the case where the mesh parameter is nullptr? While this method technically accepts a pointer none of the uses imply passing nullptr is valid. We need to know where the nullptr is coming from first.

There are already some but we need to sprinkle nullptr checks for every gvrrenderdata::mesh() now since there is a new way for renderdata::mesh_ to become null.

---
I do know this nullptr check masks some of the crashes but they are down to thread-safety issues and we should be understanding and fixing those. In fact, we should abort the program if somebody passes a nullptr. Better yet, use something better than a raw pointer to indicate that the contract is to pass a valid mesh instance always.